databuilder/databuilder/extractor/neo4j_extractor.py

```diff
@@ -65,16 +63,13 @@ def close(self) -> None:
     def _get_driver(self) -> Any:
         """
         Create a Neo4j connection to Database
+        https://github.com/neo4j/neo4j-python-driver#connection-settings-breaking-change
         """
-        trust = neo4j.TRUST_SYSTEM_CA_SIGNED_CERTIFICATES if self.conf.get_bool(Neo4jExtractor.NEO4J_VALIDATE_SSL) \
-            else neo4j.TRUST_ALL_CERTIFICATES
         return GraphDatabase.driver(uri=self.graph_url,
                                     max_connection_lifetime=self.conf.get_int(
                                         Neo4jExtractor.NEO4J_MAX_CONN_LIFE_TIME_SEC),
                                     auth=(self.conf.get_string(Neo4jExtractor.NEO4J_AUTH_USER),
-                                          self.conf.get_string(Neo4jExtractor.NEO4J_AUTH_PW)),
-                                    encrypted=self.conf.get_bool(Neo4jExtractor.NEO4J_ENCRYPTED),
-                                    trust=trust)
```

Comment:
we still need these two for people running older neo4j to be able to explicitly disable encryption or configure different certificate trust policies

Reply:
or maybe i'm wrong. See https://neo4j.com/docs/python-manual/current/client-applications/#python-driver-connection-uris